app/src/main/java/com/simpragma/recipe/recipeapp/DatabaseHandler.java

```diff
@@ -0,0 +1,127 @@
+package com.simpragma.recipe.recipeapp;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+import com.simpragma.recipe.recipeapp.ResultList;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Madvesha on 2/4/18.
+ */
+
+public class DatabaseHandler extends SQLiteOpenHelper {
```

Comment:
Where are you using room? Don't see anything related to that

Reply:
still i am pushing room to branch